<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,8,1" columnstretch="0,1" rowminimumheight="0,0,5" columnminimumwidth="0,8">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>gender</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="raceLabel">
       <property name="text">
        <string>race</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1" rowminimumheight="1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="2">
        <widget class="QLabel" name="writingLabel">
         <property name="text">
          <string>writing</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="mathLabel">
         <property name="text">
          <string>math</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="readingLabel">
         <property name="text">
          <string>reading</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="readingSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="mathSpinBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="writingSpinBox"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxRace"/>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="maleRadioButton">
         <property name="text">
          <string>male</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="femaleRadioButton">
         <property name="text">
          <string>female</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="resultsLabel">
       <property name="text">
        <string>results</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
